.travis.yml

```diff
@@ -0,0 +1,17 @@
+dist: trusty
+
+sudo: true
+
+services:
+  - docker
+
+os:
+  - linux
+
+addons:
+  apt:
+    packages:
+      - make
```

Comment:
I don't think you need `make` here, is this from some earlier attempts?

Reply:
I copied it from our other travis files where we need `make`. I will try pushing a commit without it and seeing if it will work.